packages/rest/src/router/external-express-routes.ts

```diff
@@ -42,12 +53,29 @@ export class ExternalExpressRoutes {
     this._staticRoutes.use(path, express.static(rootDir, options));
   }
 
+  public mountRouter(
+    basePath: string,
+    router: ExpressRequestHandler,
+    spec: RouterSpec = {paths: {}},
+  ) {
+    this._externalRoutes.use(basePath, router);
+
+    spec = rebaseOpenApiSpec(spec, basePath);
+    assignRouterSpec(this._specForExternalRoutes, spec);
+  }
+
   find(request: Request): ResolvedRoute {
-    return new ExternalRoute(this._staticRoutes, request.method, request.url, {
-      description: 'LoopBack static assets route',
-      'x-visibility': 'undocumented',
-      responses: {},
-    });
+    return new ExternalRoute(
+      this._externalRoutes,
+      this._staticRoutes,
+      request.method,
+      request.url,
+      {
+        description: 'LoopBack static assets route',
```

Comment:
This description is no longer true, please update it. E.g.

```suggestion
        description: 'External route or a static asset',
```